embark-ui/src/containers/EditorContainer.js

```diff
@@ -43,38 +72,115 @@ class EditorContainer extends React.Component {
   }
 
   openAsideTab(newTab) {
+   this.changeEditorWidth((this.state.currentAsideTab === '') ? -1 : 1);
     if (newTab === this.state.currentAsideTab) {
-      return this.setState({currentAsideTab: ''});
+      this.editor.handleResize();
+      return this.setState({
+        currentAsideTab: ''
+      });
     }
     this.setState({currentAsideTab: newTab});
   }
 
-  textEditorMdSize() {
-    return this.state.currentAsideTab.length ? 7 : 10;
+  changeEditorWidth(operation) {
```

Comment:
Do you think it would be a good idea to have OPERATION enum that would look like that
```
OPERATION = {
  MORE: 1,
  LESS: -1
}
```